README.md

```diff
@@ -5,3 +5,43 @@ This is the implementation of the Agreements Service [Open API specification](ht
 It is deployed as part of the Terraform AWS environment provisioning scripts in [ccs-scale-infra-services-shared](https://github.com/Crown-Commercial-Service/ccs-scale-infra-services-shared).
 
 CodeBuild project(s) exist in the Management account to build from source and deploy as a Docker image to ECR.
+
+
+## Provision the service infrastructure via Travis
+
+The main environments are provisioned automatically via Travis CI. Merges to key branches will trigger an automatic deployment to certain environments - mapped below:
+
+* `develop` branch -> `development` space
+* `main` branch -> `INT` space
```

Comment:
Could update this to reflect `release/###` branches now 👍

Reply:
Good call - updated 👍